### PR TITLE
[doc](auth)ranger doc

### PR DESCRIPTION
**Pull request by zddr**
# Proposed changes

Issue Number: close #xxx

## Problem summary

Describe your changes.

## Checklist(Required)

* [ ] Does it affect the original behavior
* [ ] Has unit tests been added
* [ ] Has document been added or modified
* [ ] Does it need to update dependencies
* [ ] Is this PR support rollback (If NO, please explain WHY)

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: docs/zh-CN/docs/lakehouse/multi-catalog/hive.md [hunk 1]
```diff
@@ -207,3 +190,92 @@ CREATE CATALOG hive PROPERTIES (
...
+连接开启 Ranger 权限校验的 Hive Metastore 需要增加配置 & 配置环境：
+1. 创建 Catalog 时增加：
+
+```sql
+"access_controller.properties.ranger.service.name" = "<the ranger servive name your hms using>",
+"access_controller.class" = "org.apache.doris.catalog.authorizer.RangerHiveAccessControllerFactory",
+```
+2. 配置所有 FE 环境：
+    
+   a. 将 HMS conf 目录下的三个 Ranger 配置文件Copy到 <doris_home>/conf 目录下。
```
Comment (morningman) [inline docs/zh-CN/docs/lakehouse/multi-catalog/hive.md:210]: 写明ranger配置文件的名称

Comment (morningman) [inline docs/zh-CN/docs/lakehouse/multi-catalog/hive.md:210]:
```suggestion
   1. 将 HMS conf 目录下的三个 Ranger 配置文件Copy到 <doris_home>/conf 目录下。
```

File: docs/zh-CN/docs/lakehouse/multi-catalog/hive.md [hunk 2]
```diff
@@ -207,3 +190,92 @@ CREATE CATALOG hive PROPERTIES (
...
+
+   b. 修改 ranger-<ranger_service_name>-security.xml 的属性,参考配置如下：
```
Comment (morningman) [inline docs/zh-CN/docs/lakehouse/multi-catalog/hive.md:212]:
```suggestion
   2. 修改 ranger-<ranger_service_name>-security.xml 的属性,参考配置如下：
```

File: docs/zh-CN/docs/lakehouse/multi-catalog/hive.md [hunk 3]
```diff
@@ -207,3 +190,92 @@ CREATE CATALOG hive PROPERTIES (
...
+```xml
```
Comment (morningman) [inline docs/zh-CN/docs/lakehouse/multi-catalog/hive.md:213]: 注意缩进

File: docs/zh-CN/docs/lakehouse/multi-catalog/hive.md [hunk 4]
```diff
@@ -207,3 +190,92 @@ CREATE CATALOG hive PROPERTIES (
...
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-stylesheet type="text/xsl" href="configuration.xsl"?>
+<configuration>
+    #缓存权限数据的目录，需要可写
```
Comment (morningman) [inline docs/zh-CN/docs/lakehouse/multi-catalog/hive.md:217]: 使用英文

File: docs/zh-CN/docs/lakehouse/multi-catalog/hive.md [hunk 5]
```diff
@@ -207,3 +190,92 @@ CREATE CATALOG hive PROPERTIES (
 | `map<KeyType, ValueType>` | `map<KeyType, ValueType>` | 暂不支持嵌套，KeyType 和 ValueType 需要为基础类型 |
 | `struct<col1: Type1, col2: Type2, ...>` | `struct<col1: Type1, col2: Type2, ...>` | 暂不支持嵌套，Type1, Type2, ... 需要为基础类型 |
 | other | unsupported | |
+
+<version since="dev">
+
+## 使用Ranger进行权限校验
+</version>
```
Comment (morningman) [inline docs/zh-CN/docs/lakehouse/multi-catalog/hive.md:197]:
```suggestion
<version since="dev"></version>
```

直接这样写吧

Comment (morningman) [reply docs/zh-CN/docs/lakehouse/multi-catalog/hive.md:197]: 这里要写明目前支持的ranger的功能，比如库、表、列。不支持加密、行权限等。

**Comment by zddr** [comment]
run buildall

**Review by morningman — APPROVED** [review]
LGTM

**Pull request merged**